lib/kino_vega_lite/chart_cell.ex

```diff
@@ -18,6 +18,7 @@ defmodule KinoVegaLite.ChartCell do
   ]
 
   @count_field "__count__"
+  @fields_with_types ["x_field", "y_field", "color_field"]
```

Comment:
Maybe `typed_fields`? In my head `fields_with_types` implies `[{field, type}]` or a map :D